test/Riok.Mapperly.Tests/Mapping/UseMapperTest.cs

```diff
@@ -306,7 +306,7 @@ public void DisabledNullableShouldWork()
                 """
                 if (source == null)
                     return default;
-                var target = new global::B(_otherMapper.ToBExternal(source.Value));
+                var target = new global::B(source.Value != null ? MapToBExternal(source.Value) : default);
```

Comment:
I think this change is a bug. `MapToBExternal` should be called on the `_otherMapper`

Reply:
Thanks for pointing out. How could we oversee this😕 I‘ll look into it.

Reply:
Fixed in https://github.com/riok/mapperly/pull/1057